compose/cli/main.py

```diff
@@ -133,6 +134,7 @@ class TopLevelCommand(DocoptCommand):
       create             Create services
       down               Stop and remove containers, networks, images, and volumes
       events             Receive real time events from containers
+      exec               Execute command in running service
```

Comment:
I would probably say "Execute a command in a running container", to be clear that it's only run in a single container.